### ISSUE TITLE
Update sponsor list and sponsor logos.

### ISSUE THREAD

**Issue by Wferr** [Paradox2102/paradox2102.github.io#15]
The sponsors on the site as far as I can tell are out of date and some with out dated images. Getting a up to date lists of the sponsors so we can refine that page and update it would be awesome.

**Comment by ghost** [comment]
It would also probably be nice to have their logos link to their websites

**Comment by ohmeow** [comment]
agreed. in addition to removing whitespace from images, can include links
as well as descriptive text. if someone can get me info I'm fine to
update.

**Comment by Wferr** [comment]
I'll work on getting a upto date list of sponsors and their levels then putting that in this issue.

**Comment by trianburner** [comment]
I gathered all of the logos we need. I made them for the weekly update
video. They are all in a folder on the Google photos account  for the team.
